web/oauth_test.go

```diff
@@ -365,6 +365,52 @@ func TestOAuthAccessToken(t *testing.T) {
 	ApiClient.ClearOAuthToken()
 }
 
+func TestAuthorizeOAuthPage(t *testing.T) {
+	th := Setup(t).InitBasic()
+	th.Login(ApiClient, th.SystemAdminUser)
+	defer th.TearDown()
+	c := &Context{
+		App: th.App,
+	}
+	th.App.UpdateConfig(func(cfg *model.Config) {
+		*cfg.ServiceSettings.EnableOAuthServiceProvider = true
+		*cfg.ServiceSettings.SiteURL = "/company/mattermost"
```

Comment:
SiteURL should be a full URL. 
`https://nowhere.com/company/mattermost`
I realize the way it is being used, it doesn't matter for this test. But it should be shown being used as it will be in the wild.

Reply:
Thanks @sbishel! Updated the site url and changed the code accordingly.